.github/workflows/linter.yml

```diff
@@ -10,11 +10,15 @@ on:
 
 jobs:
   lint:
-    name: Lint markdown file
+    name: Linter
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
       - name: Lint markdown files
         run: |
           python scripts/linter.py
+      - name: Pylint python files
```

Comment:
Can we make this a separate job? That way devs can easily see whether the Markdown or Python linter is failing

Reply:
Yeah, added it as a separate job